ONEARMY/community-platform
# Added a test about creating a how-to

Please review #626 before this PR

Test steps:
- Fill up the intro: title, tags, time, difficulty, description an caption
- Upload a cover for the intro
- Fill up the first step: title, description, caption
- Upload 2 pics for the first step
- Delete 2nd, 3rd steps
- Publish and check if the newly how-to is opened

The created how-to is cleaned up every time the test suite of `how_to.spec.ts` is executed.
